dev/README.md

```diff
@@ -0,0 +1,226 @@
+```
+
+create a network with your local quorum image
+```
+$> qctl init --consensus=istanbul --qimagefull=quorum-local --num 3
+$> qctl generate network --create
+$> qctl deploy network --wait 
+```
+
+test the block number
+```
+$> qctl geth exec quorum-node1 'eth.blockNumber'
+```
+
+#### Look inside the generated minimal config `qubernetes.generate.yaml`
+The configuration we generate above (`qubernetes/dev/istanbul-loca/qubernetes.generate.yaml`) should have the 
+`Docker_Repo_Full` field set. 
+When `Docker_Repo_Full` is set, the `Quorum_Version` field is ignored and the local container (`quorum-local`) is used instead.
+
+**example** `qubernetes/dev/istanbul-loca/qubernetes.generate.yaml`:
+```
+genesis:
+  consensus: istanbul
+  Quorum_Version: 2.7.0
+  Tm_Version: 0.10.6
+  Chain_Id: "1000"
+nodes:
+- Node_UserIdent: quorum-node1
+  Key_Dir: key1
+  quorum:
+    quorum:
+      consensus: istanbul
+      Quorum_Version: 2.7.0
+      Docker_Repo_Full: quorum-local
+    tm:
+      Name: tessera
+      Tm_Version: 0.10.6
+  geth:
+    Geth_Startup_Params: ""
+- Node_UserIdent: quorum-node2
+  Key_Dir: key2
+  quorum:
+    quorum:
+      consensus: istanbul
+      Quorum_Version: 2.7.0
+      Docker_Repo_Full: quorum-local
+    tm:
+      Name: tessera
+      Tm_Version: 0.10.6
+  geth:
+    Geth_Startup_Params: ""
+- Node_UserIdent: quorum-node3
+  Key_Dir: key3
+  quorum:
+    quorum:
+      consensus: istanbul
+      Quorum_Version: 2.7.0
+      Docker_Repo_Full: quorum-local
+    tm:
+      Name: tessera
+      Tm_Version: 0.10.6
+  geth:
+    Geth_Startup_Params: ""
+```
+
+#### <a name="step5"></a> 5. Run a Mixed Quorum Network (local container nodes, and versioned nodes)
+
+We can modify the above `qubernetes.generate.yaml` to test a mixed net where `quorum-node1` runs 2.7.0 and 
+`quorum-node2` and `quorum-node3` run our local container, by removing `Docker_Repo_Full` from `- Node_UserIdent: quorum-node1`
+
+**example mixed net** `qubernetes/dev/istanbul-loca/qubernetes.generate.yaml`:
+```
+genesis:
+  consensus: istanbul
+  Quorum_Version: 2.7.0
+  Tm_Version: 0.10.6
+  Chain_Id: "1000"
+nodes:
+- Node_UserIdent: quorum-node1
+  Key_Dir: key1
+  quorum:
+    quorum:
+      consensus: istanbul
+      Quorum_Version: 2.7.0
+    tm:
+      Name: tessera
+      Tm_Version: 0.10.6
+  geth:
+    Geth_Startup_Params: ""
+- Node_UserIdent: quorum-node2
+  Key_Dir: key2
+  quorum:
+    quorum:
+      consensus: istanbul
+      Quorum_Version: 2.7.0
+      Docker_Repo_Full: quorum-local
+    tm:
+      Name: tessera
+      Tm_Version: 0.10.6
+  geth:
+    Geth_Startup_Params: ""
+- Node_UserIdent: quorum-node3
+  Key_Dir: key3
+  quorum:
+    quorum:
+      consensus: istanbul
+      Quorum_Version: 2.7.0
+      Docker_Repo_Full: quorum-local
+    tm:
+      Name: tessera
+      Tm_Version: 0.10.6
+  geth:
+    Geth_Startup_Params: ""
+```
+
+After modifying the configuration, the network will have to be generate and deployed again
```

Comment:
does the current running network need to be stopped first before running these commands?

Reply:
It should be, yes, added the `qctl destroy network` command.